Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.12-slim
+FROM python:3.13-alpine
```

Comment:
I'm a bit confused with this diff.  In main, we're [currently using python:3.13-alpine](https://github.com/CDCgov/RecordLinker/blob/main/Dockerfile#L1).

Reply:
I don't think all the changes from main had been pulled. This has been resolved with the latest pull/push